assets/javascripts/discourse/initializers/fetch-encrypt-keys.js

```diff
@@ -13,6 +13,45 @@ import { Promise } from "rsvp";
 
 const CACHE_KEY = "discourse-encrypt-bookmark-cache";
 
+function decryptBookmarks(session, response, query, username) {
```

Comment:
Nitpick: I do not think the name of this function is very relevant as it it is doing more than just decrypting bookmarks. I'd probably call it something like `addEncryptedBookmarksFromCache`.

Reply:
Descriptive names are important 👍 
I've renamed it.